### DISCUSSION TITLE
using Automower Integration with more than 1 Automower...?

### DISCUSSION THREAD

**Discussion by BravoSierraAlpha** [Thomas55555/husqvarna_automower#99] [Q&A]
I have 3 Automowers working for me.
So far, I have not installed the Integration yet, because I guess, that updating the state every 5 minutes would be too much for 3 mowers using the API?
Would it be useful to adjust the time between updates according to the number of mowers? (eg.15 minutes for 3 mowers?)

**Comment by Thomas55555** [comment]
This shouldn't be a problem, because all mowers are polled with one request. So the limit will not exceed.
